src/workers/sync/listing-store.ts

```diff
@@ -1,39 +0,0 @@
-import ConfigStore from '../../main/config';
-import { Listing } from '../types';
-import { ListingStore } from './sync';
-
-export default function getListingStore(): ListingStore {
-  function isOldListingFormat(listing: Record<string, any>): boolean {
```

Comment:
Is the old listing format transformation no longer required?

Reply:
> When a stored listing does not comply with the desired one it simply returns null. This will make the sync process recalculate them. This is needed since there is missing info that we cannot obtain

To know if a local file is the same when moved or renamed, we need to store the files `ino` and `dev`. We could loop thru the stored listing, get the new info and update it, but it's simpler to redo it